src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -46,11 +48,10 @@ export const TransactionAmount = () => {
     dispatch(
       isEvmcParaTime
         ? paraTimesActions.fetchBalanceUsingEthPrivateKey({
-            privateKey: transactionForm.privateKey,
+            ethPrivateKey: transactionForm.ethPrivateKey,
```

Comment:
I think it would be better to only dispatch the address (slightly lower chance of private key leaking)

Reply:
In this case I will dispatch an action without payload if we want to use a selector in saga to get transactionForm data